Anchor.toml

```diff
@@ -30,6 +30,12 @@ address = "ocp4vWUzA2z2XMYJ3QhM9vWdyoyoQwAFJhRdVTbvo9E" # ocp: open_creator_prot
 [[test.validator.clone]]
 address = "6Huqrb4xxmmNA4NufYdgpmspoLmjXFd3qEfteCddLgSz" # ocp: policy (allow all)
 
+[[test.validator.clone]]
+address = "99jtJwGDfaBKXtc7kxQneAGbERGK8F5XyJWHv7qTbj9G" # global deny list for libreplex royalty enforcement
+
+[[test.validator.clone]]
```

Comment:
@madergaser I was wrong before that cloning does not work... I guess I messed up with my local somehow